### PR TITLE
[air/benchmark] Torch benchmarks for 4x4

### PR DESCRIPTION
**Pull request by richardliaw**
## Why are these changes needed?

Add benchmark data for 4x4 setup.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/ray-air/benchmarks.rst [hunk 1]
```diff
@@ -169,10 +173,36 @@ We test out the performance across different cluster sizes and data sizes.
...
+      - **Dataset**
+      - **Performance**
+      - **Command**
+    * - 4 m5.2xlarge nodes (4 workers)
+      - FashionMNIST
+      - 144.75 s (vs 154.35 s Pytorch)
+      - `python workloads/torch_benchmark.py run --num-runs 3 --num-epochs 20 --num-workers 4 --cpus-per-worker 8`
+    * - 4 g4dn.12xlarge node (16 workers)
+      - FashionMNIST
+      - TODO
```
Comment (richardliaw) [inline doc/source/ray-air/benchmarks.rst:196]: TODO

**Comment by krfricke** [comment]
Passing test for 4x4: https://buildkite.com/ray-project/release-tests-pr/builds/9986#_

**Comment by krfricke** [comment]
Smoke test run: https://buildkite.com/ray-project/release-tests-pr/builds/9992#_

**Review by krfricke — APPROVED** [review]
Benchmarks are passing on 4x4 training

**Review by simon-mo — APPROVED** [review]
stamp

**Pull request merged**